builtin/mainmenu/pkgmgr.lua

```diff
@@ -299,6 +299,18 @@ function pkgmgr.render_packagelist(render_list)
 	return table.concat(retval, ",")
 end
 
+--------------------------------------------------------------------------------
+function pkgmgr.get_modpack_description(modfolder)
+	assert(modfolder, "missing modpack folder")
+	local descf = io.open(modfolder .. DIR_DELIM .. "description.txt", "r")
+	if not descf then
+		return "No description provided."
+	end
+	local content = descf:read("*all")
+	descf:close()
+	return minetest.formspec_escape(content)
```

Comment:
Not implied by the function name. Escaping should be moved outside this function.